sources/web/datalab/polymer/modules/file-manager-factory/file-manager-factory.ts

```diff
@@ -12,38 +12,69 @@
  * the License.
  */
 
+enum FileManagerType {
+  BIG_QUERY = 'bigquery',
+  DRIVE = 'drive',
+  GITHUB = 'github',
+  JUPYTER = 'jupyter',
+  MOCK = 'mock',
+  SHARED_DRIVE = 'sharedDrive',
+}
+
+interface FileManagerConfig {
+  className: any;
```

Comment:
It looks like you are using this as a class rather than a name.

Reply:
I hadn't finished my research on how class can be represented as type. Done.